src/rest/Endpoints.ts

```diff
@@ -270,4 +270,36 @@ export default class Endpoints {
   public static GUILD_VANITY_URL(guildID: string) {
     return `/guilds/${guildID}/vanity-url`;
   }
+
+  /**
+   * `/users/@me`
+   * - GET
+   * - PATCH
+   */
+  public static CURRENT_USER() {
+    return `/users/@me`;
+  }
+  /**
+   * `/users/${userID}`
+   * - GET
+   */
+  public static USER(userID: string) {
+    return `/users/${userID}`;
+  }
+  /**
+   * `/users/@me/guilds`
+   * - GET
+   * - PATCH
+   */
+  public static CURRENT_USER_GUILDS() {
+    return `/users/@me/guilds`;
+  }
+  /**
+   * ` /users/@me/channels`
+   * - POST
+   * - GET
+   */
+  public static CREATE_DM() {
```

Comment:
This doesn't support GET anymore.
However for consistency, I would maybe rename that `USER_DM` or `SELF_USER_DM`